tests/api_resources/abstract/test_api_resource.py

```diff
@@ -101,3 +113,65 @@ def test_raise_on_incorrect_id_type(self):
         for obj in [None, 1, 3.14, dict(), list(), set(), tuple(), object()]:
             with pytest.raises(stripe.error.InvalidRequestError):
                 self.MyResource.retrieve(obj)
+
+    @pytest.fixture
+    def resource(self):
```

Comment:
nit: I would create a new resource for every test, it's not that much writing and you get to vary parameters like  `stripe_version`.